src/main/webapp/assets/js/controllers/controllers.js

```diff
@@ -258,11 +262,19 @@ canvasSupportApp.controller('addUserController', ['Friend', '$scope', '$rootScop
           } else {
             if(data.data.course_id) {
               $scope.addSuccess = true;
-              var successFullSections = $('#successFullSections').text();
-              $('#successFullSections').text(successFullSections  + ' ' + sectionName);
+              if (checkedSections === sectNumber){
+                $scope.newUser = false;
+                $scope.newUserFound = false;
+                $scope.none = false;
+                $scope.userAvailable  = false;
+                $scope.friendEmailAddress ='';
+                $scope.friend = {};
```

Comment:
I like it - but each scope is isolated. I should have made the scope settings an object ($scope.settings) so that the reset would only take one line ($scope.settings = {}) - maybe for LTI version?